private/pkg/slicesext/slicesext.go

```diff
@@ -12,36 +12,6 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// Some implementations are copied from "slices", and hence
```

Comment:
It looks like any methods from stdlib slices have been removed now so removed this notice.